ngraph/core/reference/include/ngraph/runtime/reference/prior_box_clustered.hpp

```diff
@@ -38,16 +39,11 @@ namespace ngraph
                 int64_t img_height = img[0];
 
                 // TODO: Uncomment after PriorBoxClustered is aligned with the specification.
```

Comment:
I think the operation revising should allow to align spec and op and remove this commented code.

Reply:
Removed the obsolete comment 👍